packages/rlc-codegen/src/interfaces.ts

```diff
@@ -112,10 +112,28 @@ export interface Parameter extends Schema {
 
 export interface OperationParameter {
   operationName: string;
-  parameters: ParameterMetadata[];
+  /**
+   * An operation with multiple request parameters means that 
+   * the operation can get different values for content-type and each value 
+   * may have a different type associated to it.
+   */
+  parameters: ParameterMetadatas[];
 }
+
+export interface ParameterMetadatas {
+  parameters?: ParameterMetadata[];
+  body?: ParameterBodyMetadata;
+}
+
+export interface ParameterBodyMetadata {
+  // In case of formData we'd get multiple properties in body marked as partialBody
+  isPartialBody?: boolean;
+  body?: ParameterBodySchema[];
+}
+
+export type ParameterBodySchema = Schema;
 export interface ParameterMetadata {
-  type: "query" | "path" | "header" | "body";
+  type: "query" | "path" | "header";
```

Comment:
why remove body?

Reply:
body needs extra information so I use separate model to handle it